query/src/main/java/io/keen/client/java/result/StringResult.java

```diff
@@ -31,4 +36,19 @@ public boolean isString() {
     public String stringValue() {
         return result;
     }
+
+    @Override
+    public String toString() {
+        return ReflectionToStringBuilder.toString(this, ToStringStyle.SHORT_PREFIX_STYLE);
```

Comment:
Could you tell me why you are using reflection rather than standard equals, toString etc.? We will have better performance with oldstyle: https://stackoverflow.com/questions/10912646/hashcodebuilder-and-equalsbuilder-usage-style

Reply:
I value faster development and ease of extensibility over performance in this case. In fact, in most cases customers will iterate over whole result sets / lists and map them into their domain objects prior to performing some business logic instead of relying on this implementation.

Reply:
Yeah I agree--if this is intended to be used in any perf-critical code paths, maybe the reflection-based version isn't a great idea. However, especially for `toString()` if it's just for debug output, then it doesn't much matter, and unless a perf issue is identified, I'd tend toward using the easiest method even if a slower.